www/bust01/Uvodni_stranka_webu_Domaci_Ukol.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>

```

Comment:
Asi o tom nevis, ale vsechno co je na master branch bude pristupne z internetu. Aby clovek mohl zobrazit tuhle stranku, musel by napsat dlooouuhy URL, protoze jsi ten soubor tak pojmenoval. Nechtel bys to zkratit?

Reply:
Tahle stranka neprosla kontrolou na https://validator.w3.org 
Treba ja jsem nenasel &lt;/body&gt; tag

Navic kod neni zcela naformatovany. Kdybys to mel, videl bys, kde mas chybu.

Reply:
Tenhle soubor jsi nahral pres web nebo pres commit a push?